CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 ### 🚀 New components 🚀
 
+- `flinkmetricsreceiver`: Add implementation of Flink Metric Receiver (#10121)
```

Comment:
Please move this up to the unreleased section.

Reply:
updated